sse2neon.h

```diff
@@ -2320,6 +2353,40 @@ FORCE_INLINE __m64 _mm_sad_pu8(__m64 a, __m64 b)
         vset_lane_u16(vget_lane_u64(t, 0), vdup_n_u16(0), 0));
 }
 
+// Macro: Set the flush zero bits of the MXCSR control and status register to
+// the value in unsigned 32-bit integer a. The flush zero may contain any of the
+// following flags: _MM_FLUSH_ZERO_ON or _MM_FLUSH_ZERO_OFF
+// https://software.intel.com/sites/landingpage/IntrinsicsGuide/#text=_MM_SET_FLUSH_ZERO_MODE
+FORCE_INLINE void _MM_SET_FLUSH_ZERO_MODE(unsigned int flag)
+{
+    union {
+        fpcr_bitfield field;
+#if defined(__aarch64__)
+        uint64_t value;
+#else
+        uint32_t value;
+#endif
+    } r;
+
+#if defined(__aarch64__)
+    asm volatile("mrs %0, FPCR" : "=r"(r.value)); /* read */
+#else
+    asm volatile("vmrs %0, FPSCR" : "=r"(r.value)); /* read */
+#endif
+
+    if ((flag & _MM_FLUSH_ZERO_MASK) == _MM_FLUSH_ZERO_ON) {
```

Comment:
Rewrite with branchless. That is, no if-else statement.

Reply:
The branch has been removed.